gnat2goto/driver/tree_walk.adb

```diff
@@ -3729,6 +3780,54 @@ package body Tree_Walk is
 
    end Do_Unconstrained_Array_Definition;
 
+   -------------------------
+   -- Do_Withed_Unit_Spec --
+   -------------------------
+
+   procedure Do_Withed_Unit_Spec (N : Node_Id) is
+      Unit_Name : constant String := Get_Name_String (Get_Unit_Name (N));
+   begin
+      if Defining_Entity (N) = Stand.Standard_Standard or else
+        Unit_Name = "system%s"
+      then
+         null;
+         --   At the moment Standard or System are not processed - to be done
```

Comment:
Maybe some Put_Line warning about this?
In the message not sure if 'package' or 'unit' might be more appropriate
Put_Line (Standard_Error,
                         "At the moment Standard or System library units are not processed. ");

Reply:
Done.

Reply:
I take it back: not processing standard/system libraries is not a problem unless you rely on something in them (and then you would get the appropriate error message).

Reply:
@xbauch Nitpick:
Okay. The "if then else" construct begs the question. 
To prevent another reader wondering why the null branch of the "if then else" rather than just just "if not" maybe add a clarifying line?
I'll assume the if else then construct is in case in the future we want to consider doing something with standard/system libraries.